### PR TITLE
5 new actions

### PR DESCRIPTION
**Pull request by MarlonHeiber**
* Add new ``github.add_repository_collaborator`` action which allows user to add a collaborator to repository.
* Add new ``github.check_user_repository_collaborator`` action which allows user to check if an user is a collaborator's repository.
* Add new ``github.get_repository_collaborators`` action which allows user to list the collaborators of repository.
* Add new ``github.add_update_repository_team`` action which allows user to add a team to repository.
* Add new ``github.check_team_permissions_for_repository`` action which allows user to check if a team has access to repositor

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.1.2
```
Comment (arm4b) [inline CHANGES.md]: Let's follow the https://semver.org/ and bump the minor version for the new features like this.
```suggestion
## 2.2.0
```

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the contribution!
Looks good.

Outside of semver, we'll also need to update the `pack.yaml` with the new version.

**Comment by MarlonHeiber** [comment]
Hello @armab 
All suggestions for changes have already been made
Is something missing to approve/merge this PR?

**Comment by MarlonHeiber** [comment]
> Thanks for the contribution! Looks good.
> 
> Outside of semver, we'll also need to update the `pack.yaml` with the new version.

path.yaml updated

**Pull request closed without merging**